### PR TITLE
Create Festival app

### PR DESCRIPTION
**Pull request by GrandSchtroumpf**
Setup the festival app
Copy most of the code from catalog
Solve some import import problems

### CODE REVIEW

File: apps/festival/festival/src/app/app.component.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class AppComponent {
+  constructor(
+    theme: ThemeService,
+    tunnelService: TunnelService, // Start listening on routes changes
+    icons: IconService,  // even if not used in component, keep this to load icons
+    ym: YandexMetricaService // Need to be instantiate
+  ) {
+    theme.initTheme('dark');
```
Comment (Maxstgt) [inline apps/festival/festival/src/app/app.component.ts]: default for catalog is 'light', dunno if its important

File: apps/festival/festival/src/app/marketplace/home/home.component.html [hunk 2]
```diff
@@ -0,0 +1,55 @@
+<!-- TODO(#838): Use HostBinding instead of root div -->
```
Comment (Maxstgt) [inline apps/festival/festival/src/app/marketplace/home/home.component.html]: no issue found in zenhub, remove it

File: apps/festival/festival/src/app/marketplace/title/list/list.component.html [hunk 3]
```diff
@@ -0,0 +1,283 @@
...
+  <aside fxLayout="column" class="filter">
+    <mat-card>
+      <h6>Filters</h6>
+      <p class="mat-caption">For a specific filter, multiple choices will be applied with an "or" statement (not an "and" statement).</p>
+      <mat-card-content>
+        <mat-accordion multi="true">
+          <ng-container [formGroup]="filterForm">
+
+          <!-- Production Year -->
+          <!-- TODO #1146 -->
```
Comment (Maxstgt) [inline apps/festival/festival/src/app/marketplace/title/list/list.component.html]: no issue found, remove it

File: apps/festival/festival/src/app/marketplace/title/list/list.component.html [hunk 4]
```diff
@@ -0,0 +1,283 @@
...
+            </mat-expansion-panel-header>
+            <div fxLayout="column" fxLayoutAlign="start start" class="checkbox-padding">
+              <mat-checkbox (change)="hasBudget(budget)" color="primary" *ngFor="let budget of budgetList">
+                {{ budget.label }}
+              </mat-checkbox>
+            </div>
+          </mat-expansion-panel>
+
+          <!-- Certifications -->
+          <!-- TODO #1146 -->
```
Comment (Maxstgt) [inline apps/festival/festival/src/app/marketplace/title/list/list.component.html]: same

File: libs/utils/src/index.ts [hunk 5]
```diff
@@ -11,12 +9,10 @@ export * from './lib/pipes/reverse.module';
 export * from './lib/intercom.module';
 export { FireAnalytics } from './lib/analytics/app-analytics';
 export * from './lib/common-interfaces/terms';
-export * from './lib/algolia-autocomplete/algolia-autocomplete.module';
```
Comment (Maxstgt) [inline libs/utils/src/index.ts]: if you remove it, how do you want to import?

File: apps/festival/festival/src/app/marketplace/home/home.component.html (hunk 2 above)
Comment (GrandSchtroumpf) [reply apps/festival/festival/src/app/marketplace/home/home.component.html]: The issue is that we shouldn't use the test-id for pages.

File: libs/utils/src/index.ts (hunk 5 above)
Comment (GrandSchtroumpf) [reply libs/utils/src/index.ts]: With a direct link to the module

**Pull request merged**